lib/circuitry/tasks.rb

```diff
@@ -7,10 +7,10 @@
     logger = Logger.new(STDOUT)
     logger.level = Logger::INFO
 
-    Circuitry::Provisioning.provision(logger: logger)
-  end
+    if Rake::Task.task_defined?(:environment)
+      Rake::Task[:environment].invoke
```

Comment:
IIRC, `.execute` will only run the task if it hasn't been run already and `.invoke` will run it regardless. It might be better to use execute?